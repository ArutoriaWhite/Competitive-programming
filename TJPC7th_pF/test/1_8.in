51595611 2
38182567 1
47295673 0
