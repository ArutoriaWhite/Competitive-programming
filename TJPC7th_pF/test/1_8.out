38182567 42482505
